administration/src/bp-modules/stores/StoresCSVInput.tsx

```diff
@@ -67,8 +67,12 @@ const StoresCsvInput = ({ setAcceptingStores, fields }: StoresCsvInputProps): Re
           delimiter: ',',
           encoding: 'utf-8',
         })
-      } catch {
-        showInputError('Die Datei hat ein ungültiges Spaltenformat und kann nicht gelesen werden.')
+      } catch (error) {
+        if (error instanceof Error && error.message.includes('Invalid Record Length')) {
+          showInputError('Keine gültige CSV Datei. Nicht jede Reihe enthält gleich viele Spalten.')
```

Comment:
pardon my meticulousness 🙈 but can it be useful to highlight the problem line number?
for example, I prepared 6000 entries somehow but get this error. how do I localize the problem in the file?